### ISSUE TITLE
Enable/Disable Skip Button on AppIntro2

### ISSUE THREAD

**Issue by avluis** [PaoloRotolo/AppIntro#237]
[ ] What you've tried (include code snippets, or whole files as necessary):

Updated AppIntro from 3.4.0 to 4.0.0.
Removed deprecated/implemented new methods.
Reap the benefits of a new Skip button on AppIntro2 with no way to disable it.

[ ] What you thought should happen:

AppIntro2 should include showSkipButton just like AppIntro does.

[ ] What actually happened:

Skip Button was added to AppIntro2 but there is no way to disable it without modifying the library or extending the required files.

-----

Associated PR: #241

**Comment by Andrew-Quebe** [comment]
Good idea :)

**Comment by indywidualny** [comment]
Hi. Release this fast please. It can be a small update (4.0.1) and you will include all the currently developed things later. I need this skip button really bad. I even used reflection to hide it but it's too ugly thing to do :)

**Comment by janakagamini** [comment]
I agree with @indywidualny. If a minor release could be made with this feature that would be really helpful.

**Comment by indywidualny** [comment]
@janakagamini 

There is a way to implement it right away in your app without any updates.

```
import com.github.paolorotolo.appintro.AppIntro2;

public abstract class AppIntro2Activity extends AppIntro2 {

    public void setSkipButtonEnabled(boolean skipButtonEnabled) {
        this.skipButtonEnabled = skipButtonEnabled;
    }

}
```

And then in your intro activity:

`public class IntroActivity extends AppIntro2Activity {`

**Comment by janakagamini** [comment]
@indywidualny 

Thanks, that works!

**Comment by avluis** [comment]
Here is an additional way to do this, which follows the library convention as well.

In your intro activity, add:

```java
private void showSkipButton(boolean showButton) {
    this.skipButtonEnabled = showButton;
    setButtonState(skipButton, showButton);
}
```

Then in `onCreate(Bundle savedInstanceState)` - in the same file (after adding your slides), insert:
```java
...
showSkipButton(false);
...
```

Once the library is update to include this fix, just remove the method `showSkipButton(boolean showButton)` above and leave its call in `onCreate(Bundle savedInstanceState)` since you'll be using that same method to disable the skip button.

**Comment by avluis** [comment]
Fixed by commit b457b4d
